nc-news/components/Article.jsx

```diff
@@ -3,33 +3,48 @@ import axios from "axios";
 import { Routes, Route, useParams } from "react-router-dom";
 import Button from "react-bootstrap/Button";
 import Card from "react-bootstrap/Card";
+import api from "../utils/api";
```

Comment:
You could use destructuring here, try {getArticle, getComments} here. Then you don't have to use dot notation (api.getArticle for example)